lighthouse-core/test/gather/fake-driver.js

```diff
@@ -68,6 +68,9 @@ module.exports = {
       recordsFromLogs(require('../fixtures/perflog.json'))
     );
   },
+  get devtoolsLog() {
+    return require('../fixtures/perflog.json');
```

Comment:
can we move the file to be `devtools-log.json` or whatever? The two different terms are confusing :)